<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiveDataWidget</class>
 <widget class="QWidget" name="LiveDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>271</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>343</width>
    <height>261</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>343</width>
    <height>271</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" rowspan="2">
    <layout class="QGridLayout" name="conditionsLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="10" column="0">
      <widget class="QLabel" name="uvIndex">
       <property name="text">
        <string>UV Index</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Barometer:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lblTimestamp">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Humidity:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Wind Chill:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Temperature:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Apparent Temperature:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Dew Point:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;b&gt;Current Conditions&lt;/b&gt;</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="rainRate">
       <property name="text">
        <string>Rain Rate:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Wind Direction:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="currentStormRain">
       <property name="text">
        <string>Current Storm Rain:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Wind Speed:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="lblCurrentStormStartDate">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="currentStormStart">
       <property name="text">
        <string>Current Storm Start Date:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="solarRadiation">
       <property name="text">
        <string>Solar Radiation</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="WeatherValueWidget" name="lblTemperature" native="true"/>
     </item>
     <item row="3" column="1">
      <widget class="WeatherValueWidget" name="lblApparentTemperature" native="true"/>
     </item>
     <item row="4" column="1">
      <widget class="WeatherValueWidget" name="lblWindChill" native="true"/>
     </item>
     <item row="5" column="1">
      <widget class="WeatherValueWidget" name="lblDewPoint" native="true"/>
     </item>
     <item row="6" column="1">
      <widget class="WeatherValueWidget" name="lblHumidity" native="true"/>
     </item>
     <item row="7" column="1">
      <widget class="WeatherValueWidget" name="lblBarometer" native="true"/>
     </item>
     <item row="8" column="1">
      <widget class="WeatherValueWidget" name="lblWindSpeed" native="true"/>
     </item>
     <item row="9" column="1">
      <widget class="WeatherValueWidget" name="lblWindDirection" native="true"/>
     </item>
     <item row="10" column="1">
      <widget class="WeatherValueWidget" name="lblUVIndex" native="true"/>
     </item>
     <item row="11" column="1">
      <widget class="WeatherValueWidget" name="lblSolarRadiation" native="true"/>
     </item>
     <item row="12" column="1">
      <widget class="WeatherValueWidget" name="lblRainRate" native="true"/>
     </item>
     <item row="13" column="1">
      <widget class="WeatherValueWidget" name="lblStormRain" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WeatherValueWidget</class>
   <extends>QWidget</extends>
   <header>dash_widgets/weathervaluewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
